build.gradle

```diff
@@ -24,7 +24,7 @@ ext {
     automatticAboutVersion = '1.4.0'
     automatticRestVersion = '1.0.8'
     automatticTracksVersion = '5.1.0'
-    gutenbergMobileVersion = 'v1.120.1'
+    gutenbergMobileVersion = 'v1.121.0-alpha1-test'
```

Comment:
This is for testing purposes, we will merge a new Gutenberg version in the next release.